### PR TITLE
added noncps to avoid errors during document generation - 3.x

### PR DESCRIPTION
**Pull request by jssnull**
related with: https://github.com/opendevstack/ods-jenkins-shared-library/issues/645

### CODE REVIEW

File: src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy [hunk 1]
```diff
@@ -183,12 +190,25 @@ class LeVADocumentUseCase extends DocGenUseCase {
                 documentHistory: docHistory?.getDocGenFormat() ?: []
             ]
         ]
-
+
```
Comment (clemensutschig) [inline src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy:193]: @jssnull  - please check codenArc fails - e.g. here .. spaces ...

**Comment by metmajer** [comment]
@jssnull @clemensutschig thanks for taking care, I'll bring this to the team

**Comment by clemensutschig** [comment]
20 more (prio 3) findings that need to go away

**Review by jafarre-viewnext — APPROVED** [review]

**Review by clemensutschig — APPROVED** [review]

**Comment by clemensutschig** [comment]
We also need a fwd Port to master

**Review by s2oBCN — APPROVED** [review]

**Pull request merged**